### DISCUSSION TITLE
oneClickOneOffSearchButtons.uc.js  Request

### DISCUSSION THREAD

**Discussion by xcompon3nt** [aminomancer/uc.css.js#2] [Ideas]
Hello!
If you can please add the keyboard functionality to the oneClickOneOffSearchButtons.uc.js script like in the old behaviour pre-86, that would be really awesome. I mean like with browser.urlbar.update2.disableOneOffsHorizontalKeyNavigation set to false, i.e. working up and down arrows and working left and right arrows and single press of Enter to activate search with the selected engine.
Please!

**Comment by xcompon3nt** [comment]
I saw on reddit that you have updated the script. Thank you. It's perfect!

**Comment by aminomancer** [comment]
Sorry I missed you, fortunate that I was gonna do that anyway
